.github/workflows/test-executor.yml

```diff
@@ -0,0 +1,24 @@
+name: Test Executor
+
+on:
+  pull_request:
+    paths:
+      - "client/packages/executor/**/*.ts"
+      - "client/packages/executor/**/*.json"
+
+jobs:
+  test-executor:
+    defaults:
+      run:
+        working-directory: client/packages/executor
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
```

Comment:
CI is failing because you need to build local deps and install. You can use the makefile for the local ones (in packages) and after run a yarn

Reply:
Correcto!